script/lookup-commit.sh

```diff
@@ -235,7 +235,7 @@ test notes != "$mode" || {
 			continue
 		fi
 
-		amlog="$(git -C "$GITGIT_DIR" notes --ref=refs/notes/gitster-amlog show "$commit" 2>/dev/null | head -n 1)"
+		amlog="$(git -C "$GITGIT_DIR" notes --ref=refs/notes/maintainer-amlog show "$commit" 2>/dev/null | head -n 1)"
```

Comment:
A quick `git grep amlog` will reveal that this ref is updated outside of the `gitgitgadget/gitgitgadget` code base. Therefore, changing the name here will simply break the script.